.github/workflows/integration-tests.yml

```diff
@@ -81,10 +88,17 @@ jobs:
           && cd $JKUBE_DIR \
           && git checkout "$JKUBE_REVISION"
       - name: Install JKube
-        run: cd $JKUBE_DIR && mvn -f pom.xml -B -DskipTests clean install
+        run: cd $JKUBE_DIR && mvn -f pom.xml -B -DskipTests clean install && rm -rf $JKUBE_DIR
       - name: Install and Run Integration Tests
         run: |
           cd $JKUBE_DIR \
           && JKUBE_VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec) \
           && cd .. \
           && ./mvnw -B -POpenShift,${{ matrix.suite }} clean verify -Djkube.version="$JKUBE_VERSION" -Djunit.jupiter.execution.parallel.config.fixed.parallelism=1
+      - name: Cluster Information
+        if: always()
+        run: |
+          apt list --installed
+          oc login -u system:admin
+          oc describe node localhost
+          oc login -u developer
```

Comment:
Not super critical, Won't this get blocked by a password prompt? What's the limit of stale builds on Github actions?

Reply:
It doesn't get blocked (at least in OpenShift 3.11) is there any cli argument to bypass any prompt?

Reply:
https://github.com/manusa/jkube-integration-tests/runs/485036211?check_suite_focus=true#step:9:1

Reply:
okay, ignore